ui/apps/platform/src/Containers/Collections/CollectionsTable.tsx

```diff
@@ -91,29 +81,18 @@ function CollectionsTable({
         [setSearchFilter]
     );
 
-    function onConfirmDeleteCollection() {
+    function onConfirmDeleteCollection(collection: CollectionResponse) {
```

Comment:
By passing the `collection` here instead of reading from state, we can take advantage of the fact that the type has already been narrow to be non-null.